### ISSUE TITLE
v1.0.8 fails to find a file

### ISSUE THREAD

**Issue by MrSmith33** [richardsimko/update-tag#30]
```
Error: Could not find file '/home/runner/work/_actions/_temp_a7780744-a4d6-4173-afab-24bc706d564f/_staging/richardsimko-update-tag-3875bec/node_modules/ts-api-utils/node_modules/.bin/tsc'.
```
https://github.com/MrSmith33/nihlang/actions/runs/6525016542/job/17717273518

This happened because I was using `richardsimko/update-tag@v1`, which implicitly points to `v1.0.8`.

The runner is (if you need more details, see the link above)
```
  Ubuntu
  22.04.3
  LTS
```

---

I switched to using `richardsimko/update-tag@v1.0.7` and it [works](https://github.com/MrSmith33/nihlang/actions/runs/6525051722/job/17717362312).

**Comment by kobeomseok95** [comment]
I'm going through the same situation too.

**Comment by KRASSUSS** [comment]
Same issue here

**Comment by rsenden** [comment]
Same issue

**Comment by richardsimko** [comment]
Thanks for the report, I've released 1.0.9 which should hopefully adress this. Sorry about that, oddly enough it wasn't caught when running the action locally

**Comment by richardsimko** [comment]
Actually that version was also broken but `v1.0.10` fixes the issues
